packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/NodeComponent.java

```diff
@@ -212,6 +223,23 @@ private Element doRender(VariableManager nodeVariableManager, String targetObjec
         return new Element(NodeElementProps.TYPE, nodeElementPropsBuilder.build());
     }
 
+    private CollapsingState shouldRenderChildren(String nodeId, Optional<Node> optionalPreviousNode, Optional<IDiagramEvent> optionalDiagramEvent) {
```

Comment:
Let's be coherent and name this method something like `computeCollapsingState`. The fact that we will not render the children is only the current consequence